### ISSUE TITLE
Разместить в github исходники с помощью precommit1c

### ISSUE THREAD

**Issue by EvilBeaver** [skyksandr/1c-log-to-es#4]
С помощью проекта precommit1c (https://github.com/xDrivenDevelopment/precommit1c) разместить на гитхабе исходники внешней обработки.

Зачем нужно:
- наглядно видеть код и изменения сразу в веб-интерфейсе github
- увеличение репозиториев на языке 1С. Сотрудники github.com не хотят включать подсветку синтаксиса 1С, пока не наберется достаточно большое число репозиториев на языке 1С.

Готов проконсультировать, как задействовать precommit.

**Comment by JohnyDeath** [comment]
Так вроде ж есть исходники. Только видать старой версии в TXT

**Comment by EvilBeaver** [comment]
Да, надо **bsl**, а то не заметят.

**Comment by JohnyDeath** [comment]
А чтобы был BSL нужна платформа не ниже 8.3.7 верно?

**Comment by EvilBeaver** [comment]
нет, файлы создает прекоммит. Надо спросить у @pumbaEO или @bambr1975, можно ли переключить на BSL. Заодно и @nixel2007 позовем, он тоже может что-то знать :)

**Comment by pumbaEO** [comment]
Я сейчас эксперементирую с полным отказом от бинарников и использование выгрузки/загрузки 8.3.8 , если используются только упр.формы - тогда можно поэксперементировать и с новым форматом.

**Comment by skyksandr** [comment]
В обработке на данный момент вообще нет форм, предназначалась для запуска в пакетном режиме с передачей параметров в виде Json в строку запуска.

**Comment by bambr1975** [comment]
Достаточно поменять V8Reader на тот, который к коммите https://github.com/xDrivenDevelopment/v8Reader/tree/develop

**Comment by nixel2007** [comment]
Платформа значения не имеет. Для расширения bsl просто нужен прекоммит 2.0. 
Модуль объекта так же разложится в bsl. 

@evilbeaver радетель нового расширения?) 

P.S. С учётом того, что это я просил @bambr1975 переделать v8reader на bsl, да, я могу что-то знать ;)

**Comment by EvilBeaver** [comment]
@nixel2007 я радетель подсветки синтаксиса 1С в гитхабе.
